acceptance/scripts/generic/testrun.sh

```diff
@@ -16,7 +16,7 @@ do_init()
   BEAKER_INIT="$BEAKER_INIT --helper $BEAKER_HELPER"
   BEAKER_INIT="$BEAKER_INIT --options-file $BEAKER_OPTIONS"
   BEAKER_INIT="$BEAKER_INIT --post-suite $BEAKER_POSTSUITE"
-  BEAKER_INIT="$BEAKER_INIT --load-path acceptance/lib"
+  BEAKER_INIT="$BEAKER_INIT --load-path $BEAKER_LOADPATH"
```

Comment:
Why did this change? Would we still want to default to `acceptance/lib` if `$BEAKER_LOADPATH` isn't set?

Reply:
We do, the specification for `BEAKER_LOADPATH` below in the main script already supplies the default. We were just ignoring the env var entirely when it came to actually building the beaker invocation. I think I broke this when I refactored this file a while back.